pkg/util/utils_test.go

```diff
@@ -38,24 +38,24 @@ func TestErrorGenerator(t *testing.T) {
 	f := errorGenerator(errors)
 	for i := 0; i < errors-1; i++ {
 		if err := f(); err == nil {
-			t.Fatalf("Error should have been thrown at iteration %v", i)
+			t.Fatalf("Error should have been reported at iteration %v", i)
 		}
 	}
 	if err := f(); err == nil {
-		t.Fatalf("Error should not have been thrown this call!")
+		t.Fatalf("Error should not have been reported by this call.")
 	}
 }
 
 func TestRetry(t *testing.T) {
 
 	f := errorGenerator(4)
 	if err := Retry(5, f); err != nil {
-		t.Fatalf("Error should not have been raised by retry.")
+		t.Fatalf("Error should not have been reported during retry.")
```

Comment:
Nothing is technically "wrong" but it's a matter of naming conventions and RH style guide to use as clear terms as possible, where raised can be ambiguous and thrown is aggressive.